### PR TITLE
clarify "inside" term

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#575

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/data-loss-prevention-policies.md [hunk 1]
```diff
@@ -102,6 +102,9 @@ The conditions now available can determine if:
 - Content contains a label. For more information, see the below section [Using a label as a condition in a DLP policy](#using-a-label-as-a-condition-in-a-dlp-policy).
     
 - Content is shared with people outside or inside your organization.
+
+> [!NOTE]
+> Users, who have non-guest accounts in host organization's Active Directory or Azure Active Directory tenant, are considered as people inside the organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-loss-prevention-policies.md]:
```suggestion
> Users who do not have guest accounts in a host organization's Active Directory or Azure Active Directory tenant are considered people inside the organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - suggestion

**Comment by mypil** [comment]
@MaratMussabekov  - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@chrfox - PR has been copy edited and is ready for final review. This is based on issue #575.

Thank you.

cc: @kenwith

**Pull request merged**